### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by bobmatyas**
Updates from issues backlog.

### CODE REVIEW

File: _data/betting.yml [hunk 1]
```diff
@@ -5,3 +5,10 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
+
+    - name: win2day
+      url: https://www.win2day.at/
+      img: win2day.png
+      tfa: No
+      twitter: win2dayat
+      facebook: win2dayat
```
Comment (kmpoppe) [inline _data/betting.yml]:
```suggestion
      facebook: win2dayat
      lang: de
      
```
Adding `lang: de` will translate the text on the call-to-action buttons. Additionally, the data files should end with a new line.

File: _data/banking.yml [hunk 2]
```diff
@@ -649,9 +649,10 @@ websites:
     - name: MBNA Canada
       url: https://www.mbna.ca
       img: mbnacanada.png
-      tfa: No
-      twitter: MBNA_Canada
-      facebook: MBNACanada
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      doc: https://www.mbna.ca/help-centre/credit-cards/two-step-verification-faq/index.jsp
```
Comment (kmpoppe) [inline _data/banking.yml]:
```suggestion
      doc: https://www.mbna.ca/help-centre/credit-cards/two-step-verification-faq/
```
`index.jsp` is superflous in that link.

**Review by kmpoppe — CHANGES_REQUESTED** [review]
Thank you for your contribution @bobmatyas. Please take a look at the line comments I added.
// Kai

**Comment by bobmatyas** [comment]
> Thank you for your contribution @bobmatyas. Please take a look at the line comments I added.
> // Kai

Thanks. I made the requested changes and will keep the reasons why in mind for future edits.

**Review by kmpoppe — APPROVED** [review]

**Comment by kmpoppe** [comment]
@bobmatyas Just a quick reminder. Next time, if you have multiple updates, please put them in different branches and different pull requests so that adding/reverting them easier.
Thanks!

**Comment by bobmatyas** [comment]
> @bobmatyas Just a quick reminder. Next time, if you have multiple updates, please put them in different branches and different pull requests so that adding/reverting them easier.
> Thanks!

Sorry, I missed that part in the contribution guidelines. I will keep things separate in the future.

**Pull request merged**